aws-ssm-maintenancewindow/aws-ssm-maintenancewindow.json

```diff
@@ -113,6 +113,7 @@
         },
```

Comment:
This is not included in the scope of this pr. But I was referring to this schema when I built mine the other day. And I noticed that for the cutoff and duration property, you used minLength and maxLength instead of minimum and maximum. I'm wondering why we used those?

Reply:
Actually it should be minimum and maximum seeing as the field type is integer i think overlooked it before. have corrected it.